history-of-quakers/modernized/24-chapter-22.adoc

```diff
@@ -1681,14 +1768,18 @@ Seeing their relief is to them so needful, so harmless to all,
 and so useful to the government and their neighbors;
 let us a little consider the common objections; which may be summed up in short thus:
 
-First objection, How shall we then be at a certainty?
+[.numbered]
```

Comment:
Should this be numbered like this? Especially _"First objection"_?

Reply:
Sure, I think it seems fine like this. 👍